db/seeds.rb

```diff
@@ -33,12 +33,12 @@
 Planhabit.create(plan_id: plan.id, habit_id: habit11.id)
 
 
-plan2 = Plan.create(plan_name: "Launch Prep-Standard", author: "Paul M. Thiebauth", author_info: "Web Developer - Boston, MA")
-habit12 = Habit.create(name:"Read Launcher material for 30 mins")
-habit13 = Habit.create(name:"Practice coding for 45 mins")
-habit14 = Habit.create(name:"Create 5 new Ruby/Rails flashcards")
-habit15 = Habit.create(name:"Study flashcards for 10 minutes")
-habit16 = Habit.create(name:"Write down what you learned today")
+plan2 = Plan.create(plan_name: 'Launch Prep-Standard', author: 'Paul M. Thiebauth', author_info: 'Web Developer - Boston, MA')
```

Comment:
Line is too long. [126/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.